src/engines/nwn2/store.cpp

```diff
@@ -71,6 +71,46 @@ void Store::setStoreMaximumBuyPrice(int32 max) {
 	_maxBuyPrice  = (max < 0) ? -1 : max;
 }
 
+Item *Store::getFirstItemInInventory() {
+	// Cycle through the inventory types
+	for (InventoryType it = 0; it < kInventoryTypes; it++) {
+		// Retrieve the first item in this inventory
+		Item *item = _inventory[it]->getFirstItemInInventory();
```

Comment:
Since you're filling in the `_inventory` array by reading from the GFF, you can't be certain all types are in there. So you need to check that `_inventory[it]` is != nullptr before dereferencing.

Not just here, but everywhere you're accessing parts of `_inventory`.